### PR TITLE
adds layout for news posts

### PR DESCRIPTION
**Pull request by andresgalante**
New layout for news posts called `news`. 
It has more information:
- the author's name and avatar
- date of posting
- module name and logo
- platform
- tags

### CODE REVIEW

**Comment by andresgalante** [comment]
@lfryc Can you review this one.

File: css/styles.less [hunk 1]
```diff
@@ -1440,4 +1458,21 @@ input.code {
...
+
+
+
+
+
+
+
+
+
+
```
Comment (lfryc) [inline css/styles.less]: white space? :-)

File: _includes/breadcrumbs.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+    <ol class="breadcrumb" style="margin:0">
+      <li>
+        {% if page.section == 'guides' %}
+          <a href="{{ site.baseurl }}/getstarted/guides/">Guides</a>
+        {% else if page.section == 'modules' %}
```
Comment (lfryc) [inline _includes/breadcrumbs.html]: again, we need to use `elsif` here, instead of `else if`

**Comment by lfryc** [comment]
@andresgalante except my comments look good

**Comment by abstractj** [comment]
@andresgalante landed here: https://github.com/aerogear/aerogear.org/tree/new-design

**Pull request merged**